.github/workflows/build-and-publish-chart.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         author_name: "GitHub Actions Bot"
         author_email: "actions@github.com"
         message: "Update Chart.yaml to version ${{ env.NEW_VERSION }}"
-        path: "charts/cloudzero-agent/Chart.yaml"      
```

Comment:
I'm removing `path` because I don't think that's a supported argument. see the warning here:
```
Warning: Unexpected input(s) 'path', valid inputs are ['add', 'author_name', 'author_email', 'commit', 'committer_name', 'committer_email', 'cwd', 'default_author', 'fetch', 'message', 'new_branch', 'pathspec_error_handling', 'pull', 'push', 'remove', 'tag', 'tag_push', 'github_token']
```